### PR TITLE
Toolbar extendable by button in top and menu in bottom

### PR DESCRIPTION
**Pull request by ksuess**
Fix https://github.com/plone/volto/issues/4333

### CODE REVIEW

**Comment by ksuess** [comment]
Ready for reviewing. Can you please have a look, @tiberiuichim. Thank you!

**Comment by ksuess** [comment]
Ready to merge, @sneridagh

File: docs/source/configuration/settings-reference.md [hunk 1]
```diff
@@ -339,6 +339,41 @@ apiExpanders
     ```
     The configuration accepts a list of matchers, with the ability to filter by the request path and action type for maximum flexibility.
     It also accepts a `querystring` object that allows configuring the expanders via query string parameters, such as the navigation expander.
+
+toolbarComponents
```
Comment (sneridagh) [inline docs/source/configuration/settings-reference.md:343]: @ksuess is there a typo in here? Should it be `additionalToolbarComponents`?

**Review by sneridagh — CHANGES_REQUESTED** [review]
LGTM, only the typo in docs!

File: docs/source/configuration/settings-reference.md [hunk 2]
```diff
@@ -339,6 +339,41 @@ apiExpanders
     ```
     The configuration accepts a list of matchers, with the ability to filter by the request path and action type for maximum flexibility.
     It also accepts a `querystring` object that allows configuring the expanders via query string parameters, such as the navigation expander.
+
+additionalToolbarComponents
+    For additional toolbar menues, the menu body component needs to be added to the on demand loaded components.
```
Comment (stevepiercy) [inline docs/source/configuration/settings-reference.md:344]: Spelling fixes.

```suggestion
    For additional toolbar menus, the menu body component needs to be added to the on-demand loaded components.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/source/configuration/settings-reference.md (hunk 1 above)
Comment (ksuess) [reply docs/source/configuration/settings-reference.md]: yes indeed, thanks for the hint, @sneridagh !

**Review by stevepiercy — APPROVED** [review]
LGTM!

**Pull request merged**